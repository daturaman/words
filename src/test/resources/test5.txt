6E+5 is the same as 600,000