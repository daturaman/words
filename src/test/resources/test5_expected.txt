Word count = 6
Average word length = 3.667
Number of words of length 2 is 2
Number of words of length 3 is 1
Number of words of length 4 is 2
Number of words of length 7 is 1
The most frequently occurring word length is 2, for word lengths of 2 & 4